config/routes.rb

```diff
@@ -7,11 +7,11 @@
       sessions: 'publishing_users/sessions'
     }
 
-  root to: 'tasks#index'
+  root to: 'tasks#my'
```

Comment:
There's routes for that below, and agree, on home controller we should redirect as a before action